src/commands/create.js

```diff
@@ -83,7 +84,7 @@ export default asyncCommand({
 
 		spinner.text = 'Initializing project';
 
-		await npm(target, ['init', '-y']);
+		await spawn('npm', ['init', '-y'], { cwd: target, stdio: 'ignore' })
```

Comment:
future thing to do might be to move the `which` check up and use yarn here too.  For now anyone running yarn would have npm installed anyway.